javascripts/discourse/lib/kanban-list-builders/categories.js

```diff
@@ -0,0 +1,62 @@
+import Category from "discourse/models/category";
+import Site from "discourse/models/site";
+
+export default function buildCategoryLists({ kanbanHelper, param }) {
+  const lists = [];
+
+  if (param) {
+    let categories = param
+      .split(",")
+      .map((c) => Category.findBySlug(...c.split("/").reverse()));
+    categories.filter((c) => c !== undefined);
+
+    lists.push(
+      ...categories.map((category) => {
+        return {
+          title: `${category.name}`,
```

Comment:
Do we need to `toString()` the category name?

```suggestion
          title: category.name,
```